cadl-tests/package.json

```diff
@@ -11,10 +11,8 @@
     "@cadl-lang/rest": "~0.17.0",
     "@cadl-lang/openapi": "~0.12.0",
     "@azure-tools/cadl-azure-core": "~0.7.0",
-    "@azure-tools/cadl-ranch": "latest",
```

Comment:
why remove  "@azure-tools/cadl-ranch" and "@azure-tools/cadl-ranch-api" dependency?

Reply:
They are in others dependency.

If everything works with less dependency, why add them?